docs/wip-plugins.md

```diff
@@ -0,0 +1,140 @@
+# Plugins
+
+If Chatterino is compiled with the `CHATTERINO_PLUGINS` CMake option, it can
+load and execute Lua files. Note that while there are attempts at making this
+decently safe, we cannot guarantee safety.
+
+## Plugin structure
+
+Chatterino searches for plugins in the `Plugins` directory in the app data, right next to `Settings` and `Logs`.
+
+Each plugin should have its own directory.
+
+```
+Chatterino dir/
```

Comment:
```suggestion
Chatterino Plugins dir/
```

Plugins are located inside the `Plugins` directory, not the root Chatterino directory.